test/test_integration_resource.py

```diff
@@ -154,6 +149,21 @@ def test_user_change_password_does_not_provide_password_prompts_input(self):
                                      ['user', 'change-password'])
         self.assertIn("Successfully changed password for", output)
 
+    @integration_test()
+    def test_user_change_password_does_not_provide_password_prompts_input_insecure(self):
+        # Login as user to avoid changing admin password
+        self.setup_insecure()
+        with patch('getpass.getpass', side_effect=['Mypassw0rD2\!']):
+            some_user_api_key = self.invoke_cli(self.cli_auth_params,
+                                                ['user', 'rotate-api-key', '-i', 'someuser'])
+
+            extract_api_key_from_message = some_user_api_key.split(":")[1].strip()
+            self.invoke_cli(self.cli_auth_params,
+                            ['login', '-i', 'someuser', '-p', extract_api_key_from_message])
+            output = self.invoke_cli(self.cli_auth_params,
+                                     ['user', 'change-password'])
```

Comment:
duplicate test so that we have

```
         output = self.invoke_cli(self.cli_auth_params,
                                     ['user', 'change-password', 'Mypassw0rD2\!']])
```

Reply:
for insecure